PresentScreenings/Controllers/ViewController.cs

```diff
@@ -648,35 +648,27 @@ public void ToggleAttendance(string filmfan)
 
         public void MoveScreening(bool forward)
         {
-            // Move the current screening to the nearest unattended space.
+            // Move the current screening to the nearest unattended space within the day.
             Screening screening = Plan.CurrScreening;
             if (screening is OnDemandScreening onDemandScreening)
             {
                 onDemandScreening.MoveStartTime(GetSpanToFit(onDemandScreening, forward));
-
-                // Sort the screenings on the current screen.
-                var day = Plan.CurrDay;
-                var screen = onDemandScreening.DisplayScreen;
-                var screenings = Plan.ScreenScreenings[day];
-                screenings[screen].Sort();
-
-                // Refresh the view.
-                UpdateDayAttendanceStatus(onDemandScreening);
+                Plan.InitializeDays();
                 SetCurrScreening(onDemandScreening);
-                //ReloadScreeningsView();
```

Comment:
Outside scope by far, but a logic step after avoiding overlap in inter-day moving of screenings.
Thanks!